### PR TITLE
SSH API auth

### PR DESCRIPTION
**Pull request by kellyrowland**
PR to add in the SSH API auth.

### CODE REVIEW

File: newt/settings.py [hunk 1]
```diff
@@ -144,7 +144,7 @@
 )
 
 AUTHENTICATION_BACKENDS = (
-    'django.contrib.auth.backends.ModelBackend',
+    'authnz.adapters.sshapi_backend.SSHAPIBackend',
```
Comment (shreddd) [inline newt/settings.py]: can you revert this setting? we want to leave the default settings as is

File: newt/settings.py [hunk 2]
```diff
@@ -244,8 +243,8 @@
             'models': "",
         },
         'AUTH': {
-            'adapter': 'authnz.adapters.dbauth_adapter',
-            'models': '',
+            'adapter': 'authnz.adapters.sshapi_adapter',
+            'models': 'authnz.adapters.sshapi_models',
```
Comment (shreddd) [inline newt/settings.py]: We should revert this setting since the default should use dbauth_adapter

File: newt/settings.py [hunk 3]
```diff
@@ -215,8 +215,7 @@
 NEWT_HOST = 'localhost'
 NEWT_DOMAIN = 'nersc.gov'
 NEWT_COOKIE_LIFETIME = 43200
-MYPROXY_SERVER = 'nerscca2.nersc.gov'
-
+SSHPROXY_SERVER = 'https://sshproxy.nersc.gov/create_pair/default/'
```
Comment (shreddd) [inline newt/settings.py]: I think we want to have both MYPROXY_SERVER and SSHPROXY_SERVER settings available, depending on the chosen backend. Could you include both?

**Comment by kellyrowland** [comment]
@shreddd please see the most recent commit; it should address all review points

**Pull request merged**